docs/api/formik.md

```diff
@@ -145,6 +145,10 @@ Number of times user tried to submit the form. Increases when [`handleSubmit`](#
 Set the value of a field imperatively. `field` should match the key of
 `values` you wish to update. Useful for creating custom input change handlers. Calling this will trigger validation to run if `validateOnChange` is set to `true` (which it is by default). You can also explicitly prevent/skip validation by passing a third argument as `false`.
 
+#### `setFieldValues: (values: Partial<Values>, shouldValidate?: boolean) => void`
```

Comment:
Please let me know if there are other sections of documentation I should be updating as well as this!

Reply:
docs should be updated as well.